### PR TITLE
Remove unimportant event information.

### PR DESCRIPTION
**Pull request by Syulin7**

### CODE REVIEW

File: pkg/control/service_control.go [hunk 1]
```diff
@@ -113,7 +113,6 @@ func (r RealServiceControl) createServices(namespace string, service *v1.Service
 		return nil
 	}
 	log.Infof("Controller %v created service %v", accessor.GetName(), newService.Name)
-	r.Recorder.Eventf(object, v1.EventTypeNormal, SuccessfulCreateServiceReason, "Created service: %v", newService.Name)
```
Comment (cheyang) [inline pkg/control/service_control.go:116]: How about using flags to control whether this event should be generated.

File: cmd/tf-operator.v1/app/server.go [hunk 2]
```diff
@@ -99,8 +99,8 @@ func Run(opt *options.ServerOption) error {
 	if err != nil {
 		log.Fatalf("Error building kubeconfig: %s", err.Error())
 	}
-	kcfg.QPS = 10
-	kcfg.Burst = 20
+	kcfg.QPS = 50
+	kcfg.Burst = 100
```
Comment (cheyang) [inline cmd/tf-operator.v1/app/server.go]: Why do we need to increase default QPS and Burst?

File: pkg/control/service_control.go (hunk 1 above)
Comment (Syulin7) [reply pkg/control/service_control.go:116]: Done

File: cmd/tf-operator.v1/app/server.go (hunk 2 above)
Comment (Syulin7) [reply cmd/tf-operator.v1/app/server.go]: Done

**Review by cheyang — APPROVED** [review]

**Pull request merged**